selenium_tests/cross_listing/cross_listing_base_test_case.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from urlparse import urljoin
+
+from selenium_common.base_test_case import BaseSeleniumTestCase
+from selenium_common.pin.page_objects.pin_login_page_object \
+    import PinLoginPageObject
+
+from selenium_tests.account_admin.page_objects.account_admin_dashboard_page_object \
+    import AccountAdminDashboardPage
+from selenium_tests.cross_listing.page_objects.cross_listing_index_page \
+    import IndexPagePageObject
+
+
+class CrossListingBaseTestCase(BaseSeleniumTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(CrossListingBaseTestCase, cls).setUpClass()
+
+        cls.USERNAME = settings.SELENIUM_CONFIG['selenium_username']
+        cls.PASSWORD = settings.SELENIUM_CONFIG['selenium_password']
+
+        cls.CANVAS_BASE_URL = settings.SELENIUM_CONFIG['canvas_base_url']
+        cls.test_settings = settings.SELENIUM_CONFIG['cross_listing']
+        cls.TOOL_RELATIVE_URL = cls.test_settings['relative_url']
+        cls.TOOL_URL = urljoin(cls.CANVAS_BASE_URL, cls.TOOL_RELATIVE_URL)
+
+        cls.course_info_parent_page = AccountAdminDashboardPage(cls.driver)
```

Comment:
That's what you get for thinking you can save a little time and copy and paste.  :(  
Nice catch; I'll be a little more careful about it.